frontend/src/app/acrobatics/acrobatics.component.html

```diff
@@ -0,0 +1,9 @@
+<app-toolbar [title]="'ACROBATICS.TITLE' | translate" backUrl="/choreographies"></app-toolbar>
+<div class="acrobatics">
+  <div *ngFor="let acrobatic of listOfAcrobatics" class="single-acrobatic">
```

Comment:
**suggestion:** `availableAcrobatics` or something. Including the type of a variable is a bit of an antipattern